src/js/utils/__tests__/ApplicationUtil-test.js

```diff
@@ -33,8 +33,10 @@ describe('ApplicationUtil', function () {
 
       ApplicationUtil.invokeAfterPageLoad(spy);
 
+      let elapsed = setTimeout.mock.calls[0][1];
       expect(setTimeout.mock.calls[0][0]).toEqual(spy);
-      expect(setTimeout.mock.calls[0][1]).toEqual(1000);
+      expect(elapsed).toBeGreaterThan(950);
+      expect(elapsed).toBeLessThan(1050);
```

Comment:
Hey man you could use `toBeCloseTo` found [here](https://facebook.github.io/jest/docs/api.html#content)
or clock().tick found [here](http://jasmine.github.io/2.0/introduction.html#section-Mocking_the_JavaScript_Timeout_Functions)